DMCompiler/DM/Visitors/DMVisitorExpression.cs

```diff
@@ -77,6 +81,23 @@ class DMVisitorExpression : DMASTVisitor {
         public void VisitIdentifier(DMASTIdentifier identifier) {
             var name = identifier.Identifier;
 
+            if (_scopeMode == "static")
```

Comment:
Could you prevent repeating this code by taking this outside of this `if`, and exiting early if `_scopeMode != "static"`?